modules/persistent-storage-csi-sc-disabling-cli.adoc

```diff
@@ -0,0 +1,21 @@
+// Module included in the following assemblies:
+//
+// * storage/container_storage_interface/persistent-storage-csi-sc-disable.adoc
+//
+
+:_content-type: PROCEDURE
+[id="persistent-storage-csi-sc-disabling-cli_{context}"]
+= Managing the default storage class using the CLI
+
+.Prerequisites
+* Administrator rights.
```

Comment:
> Administrator rights

Is this specific enough? I see _You have access to the cluster with `cluster-admin` privileges._ as a prereq in multiple places in the docs.

Reply:
Fixed.